docs/release-notes/version-2.x.md

```diff
@@ -3,6 +3,21 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.3.0 - 2024-09-06
+
+### Added
+
+* [#423](https://github.com/nautobot/helm-charts/pull/432) Added annotations to the Nautobot deployment.
+
+### Fixed
+
+* [#434](https://github.com/nautobot/helm-charts/pull/434) Enabled celery readiness and liveliness probes by default.
```

Comment:
Should this change be under Added or Changed?

Reply:
Moved it to changed